2-mandatory/3-financial-times.js

```diff
@@ -24,6 +48,15 @@ function titleWithFewestWords(allArticleTitles) {
 */
 function headlinesWithNumbers(allArticleTitles) {
     // TODO
+    let newArr = [];
+   for (let title of allArticleTitles) {
+     if (/[0-9]/.test(title) === true) {
```

Comment:
you could simplify this a little with a shorthand character class in regex `/\d/.test`)